charts/platform-service/templates/deployment.yaml

```diff
@@ -203,14 +220,24 @@ spec:
       - name: "{{ .Values.image.pullSecret }}"
       {{- end }}
       volumes:
-{{ if .Values.opa.enabled }}      
+{{ if .Values.opa.enabled }}
       - name: opa-istio-config
-        configMap: 
+        configMap:
           name: opa-istio-config
       - name: opa-policy
-        configMap: 
+        configMap:
           name: opa-policy
-{{ end }} 
+{{ end }}
+{{- if .Values.keyVaultSecrets }}

```

Comment:
should be moved to the keyvault chart?

Reply:
I'll test how it will works with the separate chart